NEWS.txt

```diff
@@ -58,13 +95,52 @@ EXPERIMENTAL RELEASE
       'distrib' to annotations.
       
 
- * Bug fixes
- 
+* Bug fixes
+
+  - 'comp' package-specific bug fixes:
+
+    - Comp flattening dropped conversion factors for particular 
+	  instances of assignment rules. This has been corrected. 
+	  Thanks to Matthias König for reporting this. 
+      
   - 'distrib' package-specific bug fixes:
 
     - There was unfortunately an infinite loop in the code using
       the <uncertainty> element. This has been removed.
       
+  - 'render' package-specific bug fixes:
+  
+    - Some render classes were not handled correctly by language
+      bindings. This has been fixed.
+
+EXPERIMENTAL PACKAGES
+
+* New features
+
+  - 'spatial' package-specific updates:
+  
+    - Compression in the 'spatial' package is now based on strings. 
+	  The code has been changed to reflect this. 
+
+    - Support for int and uint as DataKinds has been added to the 
+	  spatial code in line with changes in the specification. 
+
+    - A range of validation tests have been added to the spatial 
+	  code. As this is an experimental passage package still in 
+	  development, we cannot guarantee that all validation has been
+	  implemented. 
+
+ * Bug fixes
+
+  - 'spatial' package-specific bug fixes:
+  
+    - There were some memory leaks in the spatial code. 
+	  Thanks to Liam Keegan for spotting these and congratulations 
+	  for being our first GitHub user to address problems using a 
+	  Pull Request. 
+  
+
+      
```

Comment:
could add that several additional miscellaneous memory leaks and possible access violations have been solved.